packages/components/src/navigable-container/container.js

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
Thank goodness for the new files. These components were always the hardest to find because of their non-obvious prior bundling.